pages/models/regulations/governors/GovSteamEU/index.md

```diff
@@ -0,0 +1,422 @@
+| $$T_\mathrm{dp}$$               | float | s    | tdp       | Derivative time constant of the power controller                             | 0             |
+| $$T_\mathrm{en}$$               | float | s    | ten       | Electro hydraulic transducer                                                 | 0.1           |
+| $$T_\mathrm{f}$$                | float | s    | tf        | Frequency transducer time constant                                           | 0             |
+| $$T_\mathrm{fp}$$               | float | s    | tfp       | Time constant for nunmerical approximation of PID controller derivative part | 0             |
+| $$T_\mathrm{hp}$$               | float | s    | thp       | High pressure (HP) time constant of the turbine                              | 0.31          |
+| $$T_\mathrm{ip}$$               | float | s    | tip       | Integral time constant of PID controller                                     | 2             |
+| $$T_\mathrm{lp}$$               | float | s    | tlp       | Low pressure (LP) time constant of the turbine                               | 0.45          |
+| $$T_\mathrm{\omega}$$           | float | s    | tw        | Speed transducer time constant                                               | 0.02          |
+| $$T_\mathrm{p}$$                | float | s    | tp        | Power transducer time constant                                               | 0.07          |
+| $$T_\mathrm{rh}$$               | float | s    | trh       | Reheater time constant of the turbine                                        | 8             |
+| $$T_\mathrm{v\,hp}$$            | float | s    | tvhp      | Control valves servo time constant                                           | 0.1           |
+| $$T_\mathrm{v\,lp}$$            | float | s    | tvlp      | Intercept valves servo time constant                                         | 0.15          |
+
+
+> [!NOTE]
+>
+> As stated in [1], to represent a turbine without a reheater, set
+> $$T_\mathrm{rh} =$$ 0 s and $$T_\mathrm{lp} \approx$$ 0.5 s .
+
+## Variables
+
+### Inputs
+
+<div id="tbl-inputs">
+
+Table 2: Inputs
+</div>
+
+| name               | type  | unit | IEC name   | description                          |
+| :----------------- | :---- | :--- | :--------- | :----------------------------------- |
+| $$\omega$$         | float | pu   | $$\omega$$ | Rotor speed                          |
+| f                  | float | pu   | f          | Local (nodal) frequency              |
+| $$P_\mathrm{e}$$   | float | pu   | pe         | Measured electrical power generation |
+| $$P_\mathrm{ref}$$ | float | pu   | Pref       | load setpoint                        |
+
+
+### Outputs
+
+<div id="tbl-outputs">
+
+Table 3: Outputs
+</div>
+
+| name             | type  | unit | IEC name | description      |
+| :--------------- | :---- | :--- | :------- | :--------------- |
+| $$P_\mathrm{m}$$ | float | pu   | Pm       | Mechanical power |
+
+
+## Equations & algorithm  
```

Comment:
please refer to the diagram.

Reply:
I do, just below 
### Initial states

ok?

Reply:
ok